released/usecases/services_apps/sap_build_code/step1/variables.tf

```diff
@@ -0,0 +1,114 @@
+variable "globalaccount" {
+  type        = string
+  description = "The globalaccount subdomain where the sub account shall be created."
+}
+
+variable "setup_ai_launchpad" {
+  type        = bool
+  description = "Switch to enable the setup of the AI Launchpad."
+  default     = true
+}
+
+variable "subaccount_name" {
+  type        = string
+  description = "The subaccount name."
+  default     = "My SAP Build Apps subaccount."
+}
+
+variable "cli_server_url" {
+  type        = string
+  description = "The BTP CLI server URL."
+  default     = "https://cpcli.cf.sap.hana.ondemand.com"
+}
+
+variable "region" {
+  type        = string
+  description = "The region where the sub account shall be created in."
+  default     = "us10"
+
+  # Checkout https://github.com/SAP-samples/btp-service-metadata/blob/main/v0/developer/aicore.json for the latest list of regions
+  # supported by the AI Core service.
+  validation {
+    condition     = contains(["eu10-canary", "ap10", "eu10", "eu11", "jp10", "us10"], var.region)
```

Comment:
As the mission is customer facing, should we add our canary to this?

Reply:
Will remove the validation as it is an artefact of a previous tf file.